src/scaffoldmaker/meshtypes/meshtype_3d_heartarterialroot1.py

```diff
@@ -129,7 +129,7 @@ def generateBaseMesh(cls, region, options, baseCentre=[ 0.0, 0.0, 0.0 ], axisSid
             #     AnnotationGroup(region, get_heart_term("posterior cusp of aortic valve")),
             #     AnnotationGroup(region, get_heart_term("right cusp of aortic valve")),
             #     AnnotationGroup(region, get_heart_term("left cusp of aortic valve")) ]
-            cuspGroup = AnnotationGroup(region, get_heart_term("aortic valve leaflet")),
+            cuspGroup = AnnotationGroup(region, get_heart_term("aortic valve leaflet"))
```

Comment:
I noticed that there are 3 entries of "aortic valve leaflet" in Annotation Tools and selection of any of these entries will highlight all 3 leaflets on the scaffold. Is this expected?